deployment/roles/ui/tasks/jumpbox.yml

```diff
@@ -0,0 +1,4 @@
+- name: Launch container
+  shell: docker-compose up -d
```

Comment:
For the oracle process we have a separate service that is configured as so it is started automatically after the system restart. Also the admin could start and stop the service from the terminal. What is the process to start and stop the UI process on the system? Will it be automatically run after the system restart?

Reply:
I haven't thought of that.
Should we add the service in this PR or create a new issue+PR for that?

Reply:
if we introduce the UI deployment in this PR, we need to introduce the proper way to manage the process. Actually this is the reason why I asked should we introduce Molecule tests in the same PR as the UI deployment introduced.